src/astro/files/__init__.py

```diff
@@ -14,3 +16,26 @@ def check_if_connection_exists(conn_id: str) -> bool:
     """
     BaseHook.get_connection(conn_id)
     return True
+
+
+def get_file_list(path: str, conn_id: str, **kwargs) -> TaskDecorator:
```

Comment:
@pankajastro Why are we returning `TaskDecorator` shouldn't we be returning list of string/file objects?

Reply:
Not list, A task output is actually `XComArg` I'll change accordingly.